GitTfs/Core/TfsWorkspace.cs

```diff
@@ -143,11 +143,27 @@ public void Add(string path)
 
         public void Edit(string path)
         {
-            path = GetLocalPath(path);
-            _stdout.WriteLine(" edit " + path);
-            GetFromTfs(path);
-            var edited = _workspace.PendEdit(path);
-            if (edited != 1) throw new Exception("One item should have been edited, but actually edited " + edited + " items.");
+            var localPath = GetLocalPath(path);
+            _stdout.WriteLine(" edit " + localPath);
+            GetFromTfs(localPath);
+            var edited = _workspace.PendEdit(localPath);
+            if (edited != 1)
+            {
+                if (_checkinOptions.IgnoreMissingItems)
+                {
+                    _stdout.WriteLine("Warning: One item should have been edited, but actually edited " + edited + ". Ignoring item.");
+                }
+                else
+                if (edited == 0 && _checkinOptions.AddMissingItems)
```

Comment:
Can you put the `else` and `if` on the same line?

Would it make sense to do the "add missing item" check before the "ignore missing item" check?